packages/turf-boolean-intersects/test/false/Polygon/Polygon/issue-1467.geojson

```diff
@@ -0,0 +1,38 @@
+{
+  "type": "FeatureCollection",
+  "features": [
+    {
+      "type": "Feature",
+      "properties": {},
+      "geometry": {
+        "type": "Polygon",
+        "coordinates": [
+          [
+            [0, 0],
+            [0, 2],
+            [1, 2],
+            [1, 1],
+            [2, 1],
+            [2, 0],
+            [0, 0]
+          ]
+        ]
+      }
+    },
+    {
+      "type": "Feature",
+      "properties": {},
+      "geometry": {
+        "type": "Polygon",
+        "coordinates": [
+          [
+            [.1, .1],
```

Comment:
It's preferable to have 0 before the decimal point, for example pages like [this](https://github.com/Turfjs/turf/blob/a741b9c16dcdca4588abcb197d56899c8fa5d751/packages/turf-boolean-intersects/test/false/Polygon/Polygon/issue-1467.geojson)  or [this](http://geojson.io) won't be able to render the coords so it's just a extra little pain point when trying to review things quickly

Reply:
I don't understand how this file was ever committed. It is in the packages folder for a different module... I've removed it.

Reply:
And I didn't know you could preview geojson files like that, cool!